libraries/moon/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "scripts": {
+    "test": "karma start",
+    "build": "npm run test; DIST=../../docs/libraries/moon && mkdir -p $DIST && cp -r ./meta $DIST/meta && cp -r ./results $DIST/results"
+  },
+  "devDependencies": {
+    "babel-core": "^6.25.0",
+    "babel-loader": "^7.1.0",
+    "babel-preset-es2015": "^1.6.0",
```

Comment:
Getting an npm version error on this line. It says babel-preset-es2015 isn't a known version. 
I tried `^6.24.1` which is the same one being used in the Vue and it seemed to work.

Reply:
Ahh, this was because I was using `babel-preset-env` for development (Firefox was being weird with async functions), but when I switched back, it didn't seem to update. I'll fix this.